web/vim_lib.js

```diff
@@ -91,8 +91,19 @@ var LibraryVIM = {
         } 
       }
 
-      if(!handled)
-        vimjs.gui_web_handle_key(charCode || keyCode, modifiers, 0, 0);
+      if(!handled) {
+        var MAX_UTF8_BYTES = 6;
+        var chars = new Uint8Array(MAX_UTF8_BYTES + 1); // null-terminated
+        var charLen = stringToUTF8Array(String.fromCharCode(charCode), chars, 0, MAX_UTF8_BYTES);
+        if (charLen == 1) {
```

Comment:
I will look into it today. Maybe it's not necessary. Sorry for taking so long to respond.